CODEOWNERS

```diff
@@ -5,6 +5,10 @@
 # Order is important; for each modified file, the last matching
 # pattern takes the most precedence.
 
+Kconfig                                     @leandrolanzieri @jia200x @cgundogan
+*.md                                        @aabadie @jia200x
+*.murdock                                   @kaspar030
```

Comment:
```suggestion
.murdock                                   @kaspar030
```

Reply:
or would that be `/.murdock`?

Reply:
.murdock will be any .murdock file anywhere in the repo
/.murdock will only be the one in the root of the repo

Reply:
> /.murdock will only be the one in the root of the repo

that's the one I mean!

Reply:
(updated suggestion)

Reply:
Do you also want `.travis.yml`?

Reply:
nope. :) maybe @aabadie wants it!